docs/sources/source/nlp/dialogue_state_tracking.rst

```diff
@@ -264,3 +263,124 @@ References
     :style: plain
     :labelprefix: NLP-DST
     :keyprefix: nlp-dst-
+
+
+
+SGD Tutorial
+============
+
+
+Introduction
+------------
+
+One of the main building blocks of a task-oriented dialogue system that can book a table in a restaurant or order food for you is a Dialogue State Tracker (DST).
+DST should not only understand what the user just said but also remember what was said before.
+DST carries the information about what intent the user has in the conversation, for example, find a restaurant or book a plane ticket,
+and what slots along with the corresponding values were mentioned in the dialogue.
+
+
+
+This tutorial is based on the code from `examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py  <https://github.com/NVIDIA/NeMo/blob/master/examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py>`_.
+
```

Comment:
in other tutorials we were adding a note with that information at the bottom...

Reply:
moved